.github/workflows/python-package.yml

```diff
@@ -105,7 +105,7 @@ jobs:
     strategy:
       matrix:
         include:
-          - {python: '3.7', os: ubuntu-20.04, moto_server: true}
```

Comment:
Why did you do this?

Reply:
Sorry about that, I had accidently deleted this during debugging. I will re-add this on my next commit.